services/registration/src/routes/application.ts

```diff
@@ -57,6 +57,35 @@ applicationRouter.route("/").get(
   })
 );
 
+applicationRouter.route("/compile-extra-info").get(
+  checkAbility("read", "Application"),
```

Comment:
Can we change the ability check to "aggregate" instead of "read"? This should make it so only members can access this